data/remnant missions.txt

```diff
@@ -724,10 +724,11 @@ mission "Remnant: Return the Samples"
 			`	"Could you return some of them to Nasqueron? According to our logs, they were found floating in the lower levels of the atmosphere, where their natural buoyancy should keep them afloat."`
 			choice
 				`	"Sure, I would be glad to return them."`
-				`	"Sorry, I'm really not interested in braving the Archon again."`
+				`	"Sorry, I'm really not interested in dodging the Archon again."`
 					decline
 
-			`	"Thank you. Since we are continuing to work together, I'll try again." He takes a breath, concentrates, and speaks a short verse about plumage with a complex chord that sounds at once grandiose and self-deprecating. The best you can make out is "Plume." "We can't pay you for this, but if it does appease the Archon we will be able to resume our studies, and who knows where that could lead?"`
+			`	"Thank you. Since we are continuing to work together, I would like to introduce myself again." He takes a breath, thinks for a second, and speaks a short verse about plumage with a complex chord that sounds at once grandiose and self-deprecating. The best you can make out is "Plume."
```

Comment:
How about `The best you can make out is that his name is "Plume."`, because I think the current wording can still lead to some confusion.

Reply:
OK. Changed.